### PR TITLE
add support Lorenzo in DefiLlama

### PR DESCRIPTION
**Pull request by LevenR**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Lorenzo

##### Twitter Link:
https://x.com/LorenzoProtocol

##### List of audit links if any:
https://github.com/Lorenzo-Protocol/lorenzo/blob/main/audit/Lorenzo%20Protocol%20-%20Zellic%20Audit%20Report.pdf

##### Website Link:
https://www.lorenzo-protocol.xyz/

##### Logo (High resolution, will be shown with rounded borders):
Lrz-logo2 0-11

##### Chain:
Lorenzo

##### Short Description (to be shown on DefiLlama):
Lorenzo is the Bitcoin Liquidity Finance Layer, creates an efficient market in which Bitcoin holders can easily find the best opportunities to invest their unused Bitcoin liquidity and serves as the premier DeFi ecosystem in which to finance Bitcoin restaking tokens.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Liquid Staking

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/Lorenzo-Protocol

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/Lorenzo exports TVL: 
        
 
 
    bitcoin                   0
    
    total                    0

File: projects/helper/chains.json [hunk 1]
```diff
@@ -169,6 +169,7 @@
   "liquidchain",
   "litecoin",
   "loop",
+  "lorenzo",
```
Comment (g1nt0ki) [inline projects/helper/chains.json:172]: lorenzo is a chain?

File: projects/Lorenzo/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { sumTokensExport } = require("../helper/sumTokens");
+
+const LorenzoOwners = [
+    "bc1pzd6luyardlle9f7lul2y8fl72c22p6vxspc4k4g4gzgjf8975s0sr042yt",
```
Comment (g1nt0ki) [inline projects/Lorenzo/index.js:4]: what are these addresses?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

File: projects/helper/chains.json (hunk 1 above)
Comment (LevenR) [reply projects/helper/chains.json:172]: Yeah. Lorenzo is an evm compatible chain with cosmos architecture.

File: projects/Lorenzo/index.js (hunk 2 above)
Comment (LevenR) [reply projects/Lorenzo/index.js:4]: These address are mpc custody address support by cobo. Used to receive BTC staked by users, and then mint lorenzo native gas fee token stBTC to user.

File: projects/helper/chains.json (hunk 1 above)
Comment (g1nt0ki) [reply projects/helper/chains.json:172]: is the chain live? can you share rpc and explorer info?

File: projects/Lorenzo/index.js (hunk 2 above)
Comment (g1nt0ki) [reply projects/Lorenzo/index.js:4]: but there is no deposit atm?

File: projects/helper/chains.json (hunk 1 above)
Comment (LevenR) [reply projects/helper/chains.json:172]: RPC:  https://rpc.lorenzo-protocol.xyz
ChainId: 8329

File: projects/Lorenzo/index.js (hunk 2 above)
Comment (LevenR) [reply projects/Lorenzo/index.js:4]: Already have users to stake and get lorenzo native fee token(stBTC)

File: projects/helper/chains.json (hunk 1 above)
Comment (g1nt0ki) [reply projects/helper/chains.json:172]: also, any protocol already live? we usually list a chain only where there is a live defi project in it

**Comment by g1nt0ki** [comment]
merged here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/10401

**Pull request closed without merging**